GitHubSearch/Modules/Controllers/Search/SearchViewController.swift

```diff
@@ -93,7 +93,7 @@ extension SearchViewController {
         searchController.obscuresBackgroundDuringPresentation = false
         searchController.dimsBackgroundDuringPresentation     = false
         searchController.hidesNavigationBarDuringPresentation = false
-        searchController.definesPresentationContext           = true
```

Comment:
Why did you delete `searchController.`?

Reply:
I change from ```searchController.definesPresentationContext``` to ```self``` which is ur ```SearchViewController```.
And for my understanding and from the documentation [definesPresentationContext](https://developer.apple.com/documentation/uikit/uiviewcontroller#//apple_ref/occ/instp/UIViewController/definesPresentationContext)
you should set ```definesPresentationContext = true``` to tell ur ```SearchViewController``` view will be covered.

Reply:
And I found out [here](https://thomasdenney.co.uk/blog/2014/10/5/uisearchcontroller-and-definespresentationcontext/). plz check it.